nautobot/dcim/tests/test_views.py

```diff
@@ -646,24 +646,27 @@ class ManufacturerTestCase(ViewTestCases.OrganizationalObjectViewTestCase):
     @classmethod
     def setUpTestData(cls):
 
-        Manufacturer.objects.create(name="Manufacturer 1", slug="manufacturer-1")
-        Manufacturer.objects.create(name="Manufacturer 2", slug="manufacturer-2")
-        Manufacturer.objects.create(name="Manufacturer 3", slug="manufacturer-3")
-        Manufacturer.objects.create(name="Manufacturer 8")
+        manufacturer = Manufacturer.objects.first()
+
+        # FIXME(jathan): This has to be replaced with# `get_deletable_object` and
```

Comment:
This _should_ be resolved by the `get_deletable_objects` fix in #2540 as long as you have objects without protected relationships that are eligible for deletion.

Reply:
Need to address more deterministic creation of Manufacturers with and without Platforms attached, and then also DeviceTypes that line up with the Manufacturers with correct parent/child and u_height status. It's a bit of a hassle.